pkg/spdk/replica.go

```diff
@@ -1925,14 +1925,18 @@ func (r *Replica) BackupRestore(spdkClient *spdkclient.Client, backupUrl, snapsh
 			return grpcstatus.Errorf(grpccodes.Internal, err.Error())
 		}
 	} else {
+		r.log.Info("Resetting the restore for backup %v", backupUrl)
+
 		var lvolName string
 		var snapshotNameToBeRestored string
 
 		validLastRestoredBackup := r.canDoIncrementalRestore(restore, backupUrl, backupName)
 		if validLastRestoredBackup {
+			r.log.Infof("Starting an incremental restore for backup %v", backupUrl)
 			lvolName = GetReplicaSnapshotLvolName(r.Name, restore.LastRestored)
 			snapshotNameToBeRestored = restore.LastRestored
```

Comment:
Do we need to change `restore.SnapshotName` and `restore.LvolName` during incremental restore? And isn't `restore.LastRestored` a **backup** name? IIUC, DR volume will store all data into a single **snapshot** lvol.

Reply:
After discussion with @derekbit @shuo-wu , a single snapshot needs to be taken and retained for incremental restore. I will check the snapshot naming during implementation.